### PR TITLE
Fix RTL issues in onboarding flow

### PR DESCRIPTION
**Pull request by d-alleyne**
#### Proposed Changes

This commit fixes the following issues with the onboarding flow:

1. The Back and Skip buttons at the top of the page are squeezed together. 
2. The arrow for the back button is ltr and not rtl
3. Heading and subheading on theme selection page is not right aligned

Incorrect RTL alignment

These issues are solved using [CSS Logical Properties and Values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties). Instead of relying on the
`@automattic/webpack-rtl-plugin` to transform `text-align: left` to
 `text-align: right`, logical values like `start` and `end` will display element
with the correct alignment.  Similarly, positional properties like `left` and `right`
are replaced with `inset-inline-start` and `inset-inline-end`.

#### Testing

1. Set language to Arabic or Hebrew
2. Create new site at /start
3. Select domain and free plan
5. Select "Build" from the Where to start page.
6. Verify issues have been fixed.

Related to 437-gh-Automattic/i18n-issues

### CODE REVIEW

**Review by yuliyan — COMMENTED** [review]
Really nice approach!

Tested the changes and everything works as expected. Also confirmed that the new properties that you've used are well supported by our browsers support list, according to caniuse.com.

However, I don't have any prior experience with using the CSS Logical Properties and I might be missing something. So I'd prefer to wait for at least one more review before we approve and merge it.

File: client/signup/style.scss
Comment (dlind1) [inline client/signup/style.scss:450]: Running on local Calypso instance I'm getting `text-align: right;` for these selections, coming from http://calypso.localhost:3000/calypso/evergreen/entry-stepper.rtl.css with the selector being `.design-setup .design-picker .formatted-header .formatted-header__subtitle`.  It looks like there are lot of different places where `.formatted-header .formatted-header__subtitle` selector is used, should they all be updated?

Comment (dlind1) [reply client/signup/style.scss:450]: Here're the screenshots of how it looks on my end:
Screenshot 2022-06-29 at 14 35 24
Screenshot 2022-06-29 at 14 36 02

Comment (d-alleyne) [reply client/signup/style.scss:450]: The latest commit should fix this issue. Overall, this PR focuses on the issues mentioned in 437-gh-Automattic/i18n-issues. We can use this method to fix similar issues as they arise in the future, or create a new ticket where we remove our dependency on the webpack-rtl-plugin.

**Review by dlind1 — APPROVED** [review]
LGTM!

**Pull request merged**
